pkg/api/types/container_types.go

```diff
@@ -42,6 +42,13 @@ type ContainerStopOptions struct {
 	Timeout *time.Duration
 }
 
+// TopOptions specifies options for `nerdctl top`.
+type TopOptions struct {
```

Comment:
Can you rename it to `ContainerTopOptions` to be consistent with other container options?